### DISCUSSION TITLE
Monthly NTH Community Meeting?

### DISCUSSION THREAD

**Discussion by brycahta** [aws/aws-node-termination-handler#523] [Ideas]
As the maintainers of Node-Termination-Handler (NTH), we want to know if a monthly community meeting would bring value to our users and contributors.

**Comment by Omniscience619** [comment]
Count me in! It'd be a great initiative.

**Comment by brycahta** [comment]
Thanks all for the participation; you've been heard! We will be hosting community meetings starting on **Jan 4th, 2022 at 9:00AM PST** with a monthly cadence moving forward. Details can be found in the ReadMe following this [PR](https://github.com/aws/aws-node-termination-handler/pull/551) merge.
